documentation/docs/proptest/reflective_arbs.md

```diff
@@ -25,16 +28,24 @@ context("All currencies converts to EUR") { // In some spec
     converted.currency shouldBe EUR
   }
 }
+
+context("Converting to a currency and back yields the same amount") { // In some spec
```

Comment:
Can we leave in a test for a data class specifically, just in case in the future the way the reflection calls diverges.

Reply:
Dataclasses are still part of the tests. This is just the example from the documentation.

I added a second example here to illustrate that check<SomeClass>() and check(Arb.bind<SomeClass>()) are both valid.
I pushed a small change to illustrate the different use cases better.